packages/cli/src/CredentialsHelper.ts

```diff
@@ -4,52 +4,52 @@
 /* eslint-disable @typescript-eslint/no-unsafe-return */
 /* eslint-disable @typescript-eslint/no-unsafe-call */
 import { Service } from 'typedi';
-import { Credentials, NodeExecuteFunctions } from 'n8n-core';
```

Comment:
Can we please move all the `import/order` auto-fixes into a separate PR? that not only makes reviewing this easier, but also helps with bisecting for issues in the future.

Reply:
Removed from this PR